docs/core/tools/dotnet-install-script.md

```diff
@@ -1,57 +1,57 @@
 ---
 title: dotnet-install scripts
-description: Learn about the dotnet-install scripts to install the .NET Core CLI tools and the shared runtime.
-ms.date: 01/16/2019
+description: Learn about the dotnet-install scripts to install the .NET Core SDK and the shared runtime.
+ms.date: 01/06/2020
 ---
 # dotnet-install scripts reference
 
 ## Name
 
-`dotnet-install.ps1` | `dotnet-install.sh` - Script used to install the .NET Core CLI tools and the shared runtime.
+`dotnet-install.ps1` | `dotnet-install.sh` - Script used to install the .NET Core SDK and the shared runtime.
 
 ## Synopsis
 
 Windows:
 
-`dotnet-install.ps1 [-Channel] [-Version] [-InstallDir] [-Architecture] [-SharedRuntime] [-Runtime] [-DryRun] [-NoPath] [-Verbose] [-AzureFeed] [-UncachedFeed] [-NoCdn] [-FeedCredential] [-ProxyAddress] [-ProxyUseDefaultCredentials] [-SkipNonVersionedFiles] [-Help]`
+`dotnet-install.ps1 [-Channel] [-Version] [-JSonFile] [-InstallDir] [-Architecture] [-SharedRuntime] [-Runtime] [-DryRun] [-NoPath] [-Verbose] [-AzureFeed] [-UncachedFeed] [-NoCdn] [-FeedCredential] [-ProxyAddress] [-ProxyUseDefaultCredentials] [-SkipNonVersionedFiles] [-Help]`
 
-macOS/Linux:
+Linux/macOs:
 
-`dotnet-install.sh [--channel] [--version] [--install-dir] [--architecture] [--runtime] [--dry-run] [--no-path] [--verbose] [--azure-feed] [--uncached-feed] [--no-cdn] [--feed-credential] [--runtime-id] [--skip-non-versioned-files] [--help]`
+`dotnet-install.sh [--channel] [--version] [--jsonfile] [--install-dir] [--architecture] [--shared-runtime] [--runtime] [--dry-run] [--no-path] [--verbose] [--azure-feed] [--uncached-feed] [--no-cdn] [--feed-credential] [--runtime-id] [--skip-non-versioned-files] [--help]`
```

Comment:
shared-runtime is listed as obsolete. we should probably remove it from here. `--runtime` should be used instead. Also, we shuold remove it from the ps1 parameter list too.

Reply:
I have that in the notes. So just remove from the synopsis but keep it in the options list?

Reply:
@wli3 what do you think we should do here? Remove the deprecated option from the synopsis or keep it? The description for the option does say it's deprecated.

Reply:
I am not an expert on the strategy. But my vote will be removing it. Since for the install script back-compact is not a big concern. It will be no value for having the outdated field documentation.

Reply:
Yep, there is no back-compat. The option still exists but it has been deprecated in favor of another one, so there are no guarantees we won't remove it from future versions. 
We could remove it from here since another option is recommended and to avoid having folks start using that.